deployment/values.dev.yaml

```diff
@@ -83,6 +83,10 @@ rest:
     tag: 2024.09.13
 
 realtime:
+  # Override the fullname of the realtime deployment
+  # as this is needed by supabase realtime to work properly
+  # See: https://github.com/supabase-community/supabase-kubernetes/issues/74
+  fullnameOverride: "realtime-dev"
```

Comment:
Should we change this in values.cloud.yaml too?

Reply:
Yes! Would you be able to do that in separate PR since you have quick decrypt access? Thank you!

Reply:
Can do!

Reply:
Thanks so much!

Reply:
https://github.com/uw-ssec/post-disaster-comms/pull/105